gateway/rpc_storage_handler_test.go

```diff
@@ -0,0 +1,143 @@
+package gateway
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/TykTechnologies/tyk/config"
+	"github.com/TykTechnologies/tyk/storage"
+	"github.com/lonelycode/osin"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	RevokeOauthHashedToken        = "RevokeOauthHashedToken"
+	RevokeOauthToken              = "RevokeOauthToken"
+	RevokeOauthRefreshToken       = "RevokeOauthRefreshToken"
+	RevokeOauthRefreshHashedToken = "RevokeOauthRefreshHashedToken" // we do  not support hashed refresh tokens yet
+
+	DefaultOrg = "default-org-id"
+)
+
+func buildStringEvent(eventType, token, apiId string) string {
+	switch eventType {
+	case RevokeOauthHashedToken:
+		// string is as= {the-hashed-token}#hashed:{api-id}:oAuthRevokeToken
+		token = storage.HashStr(token)
+		return fmt.Sprintf("%s#hashed:%s:oAuthRevokeToken", token, apiId)
+	case RevokeOauthToken:
+		// string is as= {the-token}=:{api-id}:oAuthRevokeToken
+		return fmt.Sprintf("%s:%s:oAuthRevokeToken", token, apiId)
+	case RevokeOauthRefreshToken:
+		// string is as= {the-token}=:{api-id}:oAuthRevokeToken
+		return fmt.Sprintf("%s:%s:oAuthRevokeRefreshToken", token, apiId)
```

Comment:
The scheme on the comment is `{the-token}=:{api-id}: s:oAuthRevokeRefreshToken ` the format string should be `%s=:% s:oAuthRevokeRefreshToken ` unless if there is a typo on your comment.